zipline/src/jniTest/kotlin/app/cash/zipline/ZiplineReferenceTest.kt

```diff
@@ -41,6 +41,13 @@ internal class ZiplineReferenceTest {
     this.endpointB = endpointB
   }
 
+  @Test
+  fun referenceCanBeUsedWithoutPassingThroughZipline() {
+    val helloService = GreetingService("hello")
+    val reference = ZiplineReference<EchoService>(helloService)
+    assertThat(reference.get()).isSameInstanceAs(helloService)
```

Comment:
This is most useful for tests, where we don’t necessarily need Zipline to confirm the different components work as expected.